CBRS Small Cell Model swapped for commerical names (#945)
helium/docs: docs/5g-on-helium/cbrs-radios.mdx

[Federal Communications Commission Website](https://www.fcc.gov/35-ghz-band-overview)

5G Hotspots can participate in providing 5G cellular coverage when connected to Indoor or Outdoor
CBRS Small Cell Radios while also participating in providing LoRaWAN coverage.

:::warning

Only certified CBRS Radios provisioned with the FreedomFi firmware will operate when connected to a
5G Hotspot.

CBRS Radios that are not certified and do not use the FreedomFi firmware will not provide 5G
coverage and will not mine MOBILE.

:::

The [Helium Discord](https://discord.gg/helium) is the best place to get real-time support from the
community. The [#5g-cbrs](discord://discord.com/channels/404106811252408320/836735476659912754)
channel is most relevant for using 5G on the Network.

## Outdoor Small Cell Radios

Outdoor Small Cell Radios are more powerful than Indoor Small Cell Radios and are weatherized to
endure the elements. Outdoor Small Cell Radios create a larger area of 5G coverage while requiring a
bit more planning when installing and maintaining.

Outdoor Small Cell Radios require installation above ground level and require registration with the

> Spectrum Access System (SAS) to operate. Please be sure to evaluate your locations and ability to

Reviewer: ```suggestion
Spectrum Access System (SAS) to operate. Please be sure to evaluate your site location and ability to
```

install these Small Cell Radios, and consider hiring a professional to ensure your installation is
secure and meets all regulatory requirements.

> SAS registration is expected to be completed within 5 days of submission, Hotspot Owners will be

Reviewer: ```suggestion
SAS registration is expected to be completed within 5 days of submission. Hotspot Owners will be
```

contacted via email with a confirmation of successful registration, or with a rejection reply and an
explanation of what needs to be corrected.

## Indoor Small Cell Radios

Indoor Small Cell Radios are less powerful and are ideal for installations such as coffee shops,
bars, grocery stores, etc. Indoor Small Cell Radios are generally easier to install and create a
smaller, localized area of 5G coverage.

Indoor Small Cell Radios are physically smaller radios and can be installed at any height.

> Installing an Indoor Small Cell Radio outdoors is not recommended as they are not weather resistant.

Reviewer: ```suggestion
**Installing an Indoor Small Cell Radio outdoors is not recommended as they are not weather resistant.**
```

only theoretical as the true limiting factor is bandwidth saturation. All 5G Hotspots are built with
a 1 Gbps Network Interface Card (NIC), the throughput of all the radios combined is limited to 1
Gbps or your internet speed, whichever is lower.

## The 5G Statistics Tab on Explorer

The [Helium Explorer](https://explorer.helium.com/) will include an additional tab for all 5G
Hotspots with active Small Cell Radios, "5G Statistics"

This tab includes `Small Cell Status` and `Small Cell Location` widgets to describe the active
status and Small Cell Radio Type respectively.

Small Cell Status will be `Active` if the Small Cell has broadcast a `heartbeat` to the SAS in the
past 10 minutes. Small Cell Location will be either `Indoor` or `Outdoor` depending on the Small
Cell Radio type.

:::note 5G Hotspots with more than one Small Cell Radio

Small Cell Status will reflect all attached Small Cell Radios in aggregate, if any connected Small
Cell Radio has broadcast a `heartbeat` to the SAS, the Status will be `Active` regardless of the
actual status of any other connected Small Cell Radio.

> Refer to the 5G Hotspot dashboard for information on each connected Small Cell Radio

Reviewer: ```suggestion
Refer to the 5G Hotspot dashboard for information on each connected Small Cell Radio (local network connection only)
```

```bash
Spectrum Access status went back to “Connected” after it had been working. Wait and don’t reboot.
Suspension can last from hours to days for DPA Events. Contact support if you have been suspended for
more than 48hrs. This is more prevalent if your location is within 150km of the DPA Regions.
```

:::tip DPA Neighborhood Regions

Further information and maps of the DPA Neighborhoods in The United States and territories can be
found here:
[CBRS Encumbrances Overview](https://winnf.memberclicks.net/assets/work_products/WINNF-20-IN-0065r1%20CBRS%20Encumbrances%20Overview.pdf)

This link and the included information are for reference only and may not address all current DPA
Neighborhoods.

:::

### Small Cell Radios Need GPS

All compatible CBRS Small Cell Radios use GPS for network timing data and to assert the Small Cell
Radio's location when requesting approval to transmit from the FCC-regulated Spectrum Access System
(SAS).

Small Cell Radios will need to be placed somewhere that allows the GPS antenna to be in a location
with good GPS reception (i.e. near a window). Indoor Small Cell Radios are shipped with an external

> GPS antenna that can be positioned indipendently while Outdoor Small Cell Radios have the GPS antena

Reviewer: ```suggestion
GPS antenna that can be positioned independently while Outdoor Small Cell Radios have the GPS antenna
```
